soh/soh/Enhancements/debugconsole.cpp

```diff
@@ -1324,6 +1333,26 @@ static bool SfxHandler(std::shared_ptr<LUS::Console> Console, const std::vector<
 
     return 0;
 }
+static bool NeofetchHandler(std::shared_ptr<LUS::Console> Console, const std::vector<std::string>& args, std::string* output) {
+#ifdef __linux__
```

Comment:
Although not common, Windows and Mac are able to launch `neofetch`, although they would rely on unusual pack managers (`brew` for Mac and `chocolatey` or `winget` for Windows).